aws/sdk/integration-tests/s3/tests/throughput-timeout.rs

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+use aws_credential_types::provider::SharedCredentialsProvider;
+use aws_credential_types::Credentials;
+use aws_sdk_sts::error::DisplayErrorContext;
+use aws_smithy_async::rt::sleep::{default_async_sleep, AsyncSleep};
+use aws_smithy_async::test_util::instant_time_and_sleep;
+use aws_smithy_async::time::SystemTimeSource;
+use aws_smithy_http::body::SdkBody;
+use aws_smithy_http::byte_stream::ByteStream;
+use aws_smithy_runtime::client::http::body::minimum_throughput::MinimumThroughputBody;
+use aws_smithy_runtime::client::http::test_util::wire::{ReplayedEvent, WireMockServer};
+use aws_smithy_runtime::test_util::capture_test_logs::capture_test_logs;
+use aws_smithy_runtime::{ev, match_events};
+use aws_smithy_types::retry::RetryConfig;
+use aws_smithy_types::timeout::TimeoutConfig;
+use aws_types::region::Region;
+use aws_types::SdkConfig;
+use bytes::Bytes;
+use once_cell::sync::Lazy;
+use std::convert::Infallible;
+use std::time::{Duration, UNIX_EPOCH};
+
+#[should_panic = "minimum throughput was specified at 2 B/s, but throughput of 1.5 B/s was observed"]
+#[tokio::test]
+async fn test_throughput_timeout_less_than() {
```

Comment:
Seems like these tests should be in minimum_throughput.rs. Doesn't look like they have anything to do with S3.